manifests/server.pp

```diff
@@ -127,6 +127,7 @@
   $manage_pg_hba_conf         = $postgresql::params::manage_pg_hba_conf,
   $manage_pg_ident_conf       = $postgresql::params::manage_pg_ident_conf,
   $manage_recovery_conf       = $postgresql::params::manage_recovery_conf,
+  $manage_selinux             = $postgresql::params::manage_selinux,
```

Comment:
You can add a datatype to this:
```suggestion
  Boolean $manage_selinux     = $postgresql::params::manage_selinux,
```

Reply:
I've added the data type to this parameter.